### PR TITLE
add install command to test_create

### PR DESCRIPTION
**Pull request by kalefranz**

### CODE REVIEW

**Comment by kalefranz** [comment]
@mcg1969 see second commit here...

**Comment by kalefranz** [comment]
merging this so I can see how it does with appveyor...  but `tests/test_create.py` is still definitely WIP

File: tests/test_create.py [hunk 1]
```diff
@@ -50,25 +51,47 @@ def make_temp_env(*packages):
...
+def test_dash_c_usage_replacing_python():
+    with make_temp_env("-c conda-forge python=3.5") as env_path:
+        assert exists(join(env_path, bindir, 'python3.5'))
+        install_in_env(env_path, "decorator")
+        # TODO @mcg1969: now what?
 
 
 @pytest.mark.timeout(600)
 def test_python2_anaconda():
     with make_temp_env("python=2 anaconda") as env_path:
```
Comment (mcg1969) [inline tests/test_create.py]: `anaconda` pulls a ton of stuff. I'd recommend something simpler; say `python=2 pandas` and then test for python and numpy.

Comment (mcg1969) [inline tests/test_create.py]: At least, I wouldn't do anaconda for *every* test. In fact, `conda install anaconda` is one of the slower solver tests.

**Pull request merged**